ACPs/194-streaming-asynchronous-execution/README.md

```diff
@@ -249,22 +326,24 @@ by maximizing $\Sigma_{\forall i} (g_L - g_U)_i$ to maximize $x_W - x_A$.
 > [!TIP]
 > Although $D$ shadows a variable in ACP-176, that one is very different to anything here so there won't be confusion.
 
-Recall that the proportionality $p$ of capacity per second and target gas per second ($R = pT$) results in increasing excess such that
+Recall that the proportionality of capacity per second and target gas per second ($R = 2 \cdot T$) results in increasing excess such that
 
 $$
-x := x + \frac{g (p-1)}{p}.
+x := x + \frac{g}{2}
 $$
 
-If we allow the queue to have a bandwidth of $R\lambda^{-1}$ gas/second and limit its length $\tau$ seconds or a back-pressure window of $\omega = \tau R \lambda^{-1}$ gas, we can derive an upper bound on the difference in the changes to worst-case and actual gas excess:
+Since we limit the size of the queue to $\omega$, we can derive an upper bound on the difference in the changes to worst-case and actual gas excess:
```

Comment:
We should probably add an additional section around how the size of the queue ensures that the block executor is not constrained by the queue size (this will also likely change the maximum size of the queue, as currently the block executor _can_ be bottlenecked by the queue)

Reply:
Will leave that as a followup

Reply:
> currently the block executor _can_ be bottlenecked by the queue

How so? When $\tau \ge d$ and under full load, there is guaranteed to be at $d$ seconds' worth of chargeable gas in the queue because $\frac{\omega}{\lambda} = R\tau \ge Rd$, which is the exact execution capacity.

Reply:
The maximum size of the queue needs to be increased by the maximum size of the block (because gas isn't continuously drained from the queue, it is removed block-by-block)